openstack/compute/v2/flavors/requests.go

```diff
@@ -21,22 +21,12 @@ type ListOptsBuilder interface {
 //
 // If no IsPublic argument is supplied in the request, OpenStack treats the FlavorType as Project.
 type FlavorType string
+
 const (
-        Project FlavorType =  "true"
-        Private FlavorType = "false"
-        All FlavorType = "None"
+	Project FlavorType = "true"
```

Comment:
Okay, I think at this point I'm just waiting for @jtopjian to vote on the matter.